### PR TITLE
Update with mPMT tag and G4.10.1

### PR DESCRIPTION
**Pull request by tfeusels**
Update the nuPRISM mPMT branch with:
- all fixes to current WCSim release v.1.7.0 (including G4.10.1)
- Adding mPMT tag to ROOT output (both mPMT ID and mPMT-pmt ID) for easier fiTQun tuning of individual PMTs.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (cvilelahep) [inline CMakeLists.txt:1]: Delete remnant of conflict resolution.

**Comment by cvilelahep** [comment]
I had to substitute all `pi` with `CLHEP::pi` to get this to compile.

Once compiled it seems to run with the default (Super-K) geometry, but segfaults on first event if either `nuPRISM` or `nuPRISM_mPMT` geometries are used.

File: src/WCSimRunAction.cc [hunk 2]
```diff
@@ -155,14 +155,14 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
   flatfile->SetCompressionLevel(2); //default is 1 (minimal compression)
   masterTree = new TTree("MasterTree","Main WCSim Tree");
   if(wcsimdetector->GetIsNuPrism()){
-    if(GetSaveRooTracker())
+    if(GetSaveRooTracker()){
```
Comment (cvilelahep) [inline src/WCSimRunAction.cc:158]: This if block produces a segmentation fault it the SaveRooTracker option is turned off (e.g., if running particle guns), as `geomTree` doesn't get initialized.

Comment (mscott201) [reply src/WCSimRunAction.cc:158]: I think you need to add another "else geomTree = new TTree("Geometry","Geometry Tree");" line after the closing brace on line 165

**Pull request merged**